java/mykola-melnyk/src/com/zoolatech/lecture1/tasks/_1/Main.java

```diff
@@ -0,0 +1,20 @@
+package com.zoolatech.lecture1.tasks._1;
+
+/**
+ * Write a program that accepts numbers of miles as a floating point number,
+ * converts it to kilometers and outputs the result.
+ */
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("Enter distance in miles:");
+        Scanner scanner = new Scanner(System.in);
+        int i = 1;
```

Comment:
it seems the variable `i` is unused

Reply:
Resolved: variable i removed.
Thank you!